controllers/scripts/create_pod_status_patch.py

```diff
@@ -82,11 +85,14 @@ def get_image_version(image):
 
 def execute(cmd):
 
-    logging.debug(f"Executing command: {cmd}")
-    return_value = os.system(cmd)
-
-    if return_value != 0:
-        raise OSError(f"Execution Failed - command: {cmd}")
+    try:
+        completed_process = subprocess.run([cmd], shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+        logging.debug(f"Execution: {completed_process.stdout} - completed")
```

Comment:
I tried running this code. And got theses logs:

DEBUG:Execution: b"dd: error writing '/workdir/block-volumes/ns1': No space left on device\n5121+0 records in\n5120+0 records out\n5368709120 bytes (5.4 GB, 5.0 GiB) copied, 15.5427 s, 345 MB/s\n" - completed

INFO:pod-name: aerocluster-1-0 Finished Successfully: dd if=/dev/zero of=/workdir/block-volumes/ns1 bs=1M

DEBUG:Execution: b"dd: error writing '/workdir/block-volumes/ns': No space left on device\n5121+0 records in\n5120+0 records out\n5368709120 bytes (5.4 GB, 5.0 GiB) copied, 15.8796 s, 338 MB/s\n" - completed

I guess line 90 debug print is saying completed along with the error message(which seems to be contradictory).

And I did not see line 95 debug print in my logs.

Reply:
Fixed - now it should be ok.